scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/ssl/CustomClientHttpRequestFactory.java

```diff
@@ -0,0 +1,46 @@
+package org.cloudfoundry.autoscaler.scheduler.ssl;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.security.KeyManagementException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+
+import org.apache.http.client.HttpClient;
+import org.apache.http.conn.ssl.NoopHostnameVerifier;
+import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
+import org.apache.http.conn.ssl.TrustSelfSignedStrategy;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.ssl.SSLContextBuilder;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+
+public class CustomClientHttpRequestFactory extends HttpComponentsClientHttpRequestFactory {
```

Comment:
why?
We should just support HTTPS which means we just configure application.properties and that is it.